app/frontend/src/components/minesoperatorscreening/admin/inspection/StatusTable.vue

```diff
@@ -12,7 +12,15 @@
       item-key="inspectionStatusId"
       class="status-table"
     >
-      <template v-slot:item.createdAt="{ item }">{{ formatDateTime(item.createdAt) }}</template>
+      <template v-slot:item.createdAt="{ item }">
+        <v-tooltip bottom :disabled="item.code.toUpperCase() === 'SUBMITTED'">
```

Comment:
Wonder if the tooltip being mounted at the top would be better in this case for visibility? Can go either way - but the tooltip occludes some information either way. Perhaps to the side?

Reply:
Yeah not sure so just went with bottom, can see if a different UI is needed eventually anyways.